src/plugins/saved_objects/public/plugin.ts

```diff
@@ -10,17 +10,28 @@ import { CoreStart, Plugin } from '@kbn/core/public';
 
 import './index.scss';
 import { DataPublicPluginStart } from '@kbn/data-plugin/public';
-import { DataViewsPublicPluginStart } from '@kbn/data-views-plugin/public';
+import {
+  DataViewsPublicPluginStart,
+  SavedObjectsClientPublicToCommon,
+} from '@kbn/data-views-plugin/public';
+import {
+  getAllowedTypes,
+  getSavedObjectLabel,
+  SavedObjectManagementTypeInfo,
+} from '@kbn/saved-objects-management-plugin/public';
+import { SavedObjectsTaggingApi } from '@kbn/saved-objects-tagging-oss-plugin/public';
```

Comment:
I see the intent here (being able to use the SOT API for the finder component), but:

So, the `SOT_OSS` plugin has a `requiredBundle` and `optionalPlugin` dependency on the `SO` plugin, but the `SO` plugin imports types from the `SOT_OSS` plugin? This will be a cross/cyclic-dependency issue when moving to packages.

For the record, this is why we had to go with a inversion of control regarding the tag decorator registration, here

https://github.com/elastic/kibana/blob/3730dd0779ed8efd74aee88f57422781ec1ac122/src/plugins/saved_objects_tagging_oss/public/plugin.ts#L30

With a 'generic' decorator whose type is owned by the `SO` and not the `SOT`, plugin.

Overall, I think our best option might be to extract the finder component outside of the SO plugin, for it to be able to depend on both SO and SOT...